riscv/insns/sspush_x1.h

```diff
@@ -0,0 +1,8 @@
+#include "zicfiss.h"
+require_extension(EXT_ZICFISS);
+
```

Comment:
Is the require_extension(EXT_ZICFISS) required? If extension is not present the instructions turn into mops and that is already handled by xSSE? The require_extension will cause an illegal-instruction to be thrown...

Reply:
I think you are right ! xSSE() has already implied whether B is enabled.
I removed `require_extension(EXT_ZICFISS)`
but I think we should also add include "mop_rr_N.h"
IIt's clearer for using Zimop, although it used redundant WRITE_RD(0)."

```
if (xSSE()) {
  reg_t ssp = STATE.ssp->read();
  store_to_ssp(ssp - xlen / 8, RS2);
  STATE.ssp->write(ssp - xlen / 8);
} else 
  #include "mop_rr_N.h"
```